README.skroutz.md

```diff
@@ -0,0 +1,69 @@
+General Info
+============
+Please refer to the README.md for general information regarding resque-pool
+
+How to use
+-----------
+
+## Skroutz
+
+### YAML file config
+The `config/resque-pool.yml` can recognise whether workers of a queue should
+be forking or non-forking. For that reason the declaration of worker count
+and fork mode has been enhanced and now the following syntaxes are supported:
+
+1. Upstream's syntax
+
+```
+foo: 1
+
+production:
+    "bar,baz": 2
+    "koko": 4
+```
+
+2. Skroutz syntax
+
+```
+foo:
+  - workers: 1
+
+production:
+  "bar,baz":
+    - workers: 2
+  "koko":
+    - workers: 4
+    - fork: false
+```
+
+3. Mixed syntax
+
+```
+foo: 1
+
+production:
+  "bar,baz": 2
+  "koko":
+    - workers: 4
+    - fork: false
```

Comment:
`fork_per_job`

Ditto for the rest of this file.

Reply:
Done.